src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -881,5 +884,38 @@ object Helpers extends SparkSessionWrapper {
 
   }
 
+  def pipReport(etlDB: String = "overwatch_etl", detailed: Boolean = false):Dataset[Row] = {
+    try{
+      spark.catalog.getTable(s"${etlDB}.pipeline_report")
+      logger.log(Level.INFO, s"Overwatch has being deployed with  ${etlDB}")
+    }catch {
+      case e: Exception =>
+        val msg = s"Overwatch has not been deployed with  ${etlDB}"
+        logger.log(Level.ERROR, msg)
+        throw new BadConfigException(msg)
+    }
+
+    val pipReport = if (detailed) {
+      println("Report Extracted from pipeline_report")
+      table(s"${etlDB}.pipeline_report")
+    }else{
+      println("Report Extracted from pipReport")
+      table(s"${etlDB}.pipReport")
+    }
 
+    pipReport
+      .orderBy('Pipeline_SnapTS.desc,'moduleID)
+      .withColumn("fromTSt", from_unixtime('fromTS.cast("double") / lit(1000)).cast("timestamp"))
+      .withColumn("untilTSt", from_unixtime('untilTS.cast("double") / lit(1000)).cast("timestamp"))
+
+
+  }
+
+
+  def pipReport(etlDB: String = "overwatch_etl", orgId: String = "", detailed: Boolean = false):Dataset[Row] = {
```

Comment:
you cannot have two functions that implement default parameters. This wouldn't event compile. Always compile and ensure tests pass before submitting PR for review.